safehttp/form.go

```diff
@@ -0,0 +1,229 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package safehttp
+
+import (
+	"fmt"
+	"mime/multipart"
+	"strconv"
+)
+
+// Form contains parsed data either from URL's query or form parameters, part of
+// the body of POST, PATCH or PUT requests that are not  multipart requests. The
+// form values will only be available after parsing the form, and only through
+// the getter functions.
+type Form struct {
+	values map[string][]string
+	err    error
+}
+
+// Int64 checks whether key paramKey maps to any query or form parameter
+// values. In case it does, it will try to convert the first value to a 64-bit
+// integer and return it. If there are no values associated with paramKey, it
+// will return the default value. If the first value is not an integer, it will
+// return the default value and set the Form error field.
+func (f *Form) Int64(paramKey string, defaultValue int64) int64 {
```

Comment:
Simplify to: "param" and "default". Here and all other functions.

Reply:
Plasae fix.

Reply:
`default` is a keyword in go so we are going to stick to `defaultVal`